src/main/java/com/coveros/selenified/element/check/Equals.java

```diff
@@ -35,9 +35,11 @@
  *
  * @author Max Saperstone
  * @version 3.2.0
- * @lastupdate 3/19/2019
+ * @lastupdate 6/25/2019
  */
-public interface Equals extends Check {
+abstract class Equals extends Check {
+
+    private final static String WITH_VALUE = " with a value of <b>";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)